includes/class-wc-payments-customer-service.php

```diff
@@ -64,6 +64,54 @@ class WC_Payments_Customer_Service {
 	public function __construct( WC_Payments_API_Client $payments_api_client, WC_Payments_Account $account ) {
 		$this->payments_api_client = $payments_api_client;
 		$this->account             = $account;
+
+		add_action( 'admin_init', [ $this, 'maybe_handle_routes' ] );
```

Comment:
We can slightly simplify the PR by adding:
```php
'customer_url' => get_admin_url( null, 'admin.php?page=wc-admin&filter=single_customer&customers='. $order->get_customer_id() .'&path=/customers')
```
in file `includes/wc-payment-api/class-wc-payments-api-client.php:1022`.

Then we don't need other PHP changes and JS changes can be slightly compacted.

Reply:
Also using `admin_url( add_query_arg(...))` as in `redirect_to_single_customer` method will be great, hardcoded string as in my previous comment looks ugly =)

Reply:
Thanks! Nice suggestion. Implemented.